lib/middleware/redirects.js

```diff
@@ -34,6 +34,13 @@ var Redirect = function(source, destination, type) {
 };
 
 Redirect.prototype.test = function(url) {
+  var qs = '';
```

Comment:
We don't need to - it's `pathToRegexp` that falls apart with a query string, everywhere else we use `minimatch` which handles these things just fine:

    minimatch('/foo/bar?baz=qux', '/foo/*'); // => true